recipes/_web_packages.rb

```diff
@@ -19,30 +19,28 @@
 
 package Array(node['graphite']['system_packages'])
 
-python_package 'django' do
+pyenv_pip 'django' do
+  virtualenv node['graphite']['base_dir']
```

Comment:
If we have a library helper we should use that in the resource instead of using node attributes.

Reply:
I'm kind of unclear on what you're asking, are you saying setting the virtualenv as a library helper and then remove setting all the `virtualenv` properties when using the `pyenv_pip` resources? Or are you saying using use a library helper to get the node attribute?